frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -342,18 +249,95 @@ export const PrometheusMultilineUtilizationItem = withDashboardResources<
   },
 );
 
-export const UtilizationCard = () => {
+const UtilizationCardNodeFilter: React.FC<UtilizationCardNodeFilterProps> = ({
+  machineConfigPools,
+  onNodeSelect,
+  selectedNodes,
+}) => {
   const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const sortedMCPs = machineConfigPools.sort((a, b) => {
+    const order = ['worker', 'master'];
+    const indexA = order.indexOf(a.metadata.name);
+    const indexB = order.indexOf(b.metadata.name);
+    if (indexA === -1 && indexB === -1) {
+      return indexA - indexB;
```

Comment:
```suggestion
      return a.metadata.name - b.metadata.name;
```

Reply:
<img width="823" alt="Screen Shot 2021-06-25 at 2 07 56 PM" src="https://user-images.githubusercontent.com/895728/123467680-c6c16e00-d5be-11eb-8a0e-f9f5b330a6ce.png">

The linter does like this change.  Not sure how to fix.

Reply:
Sorry, it should be

```ts
return a.metadata.name.localeCompare(b.metadata.name);
```

Reply:
Fixed. Thanks, @spadgett